docs/topics/extraction.md

```diff
@@ -1,45 +1,79 @@
 # Extraction of Salt core modules
 
-Below are some rough steps that extract an existing set of modules into an extension while preserving the Git history. Let's use the `stalekey` engine as an example.
+## Scripted example
+Based on the [manual example below](manual-extraction-example), a new tool named [saltext-migrate](https://github.com/salt-extensions/salt-extension-migrate) was created.
 
-## A module extraction example
+Let's use the same example module (`stalekey`).
 
-### 1. Create a virtualenv and activate it
+### 1. Install `saltext-migrate` and `git-filter-repo`
 
-To create the virtualenv it is recommended to use the same Python version (MAJOR.MINOR) as the one [listed here](https://github.com/saltstack/salt/blob/master/cicd/shared-gh-workflows-context.yml).
+:::{tab} pipx
+```bash
+pip install git-filter-repo
+pipx install git+https://github.com/salt-extensions/salt-extension-migrate
+```
+:::
+:::{tab} pip
+```bash
+pip install git-filter-repo git+https://github.com/salt-extensions/salt-extension-migrate
```

Comment:
Where (venv?) that is going to be installed?

Reply:
We could instruct people to create a venv beforehand, but it might detract from the main point since it doesn't really matter for Saltext extraction/development.